Follow up PR for Gaffer VDB support

donboie opened the pull request:
- suppressed boost python registration warnings when importing pyopenvdb
- rename 'gridName' plug to 'grid'
- filter has to specified for 'LevelSetOffset', 'LevelSetToMesh' & 'MeshToLevelSet' nodes
- default level set name changed from 'levelset' to 'surface'

johnhaddon commented on python/GafferVDB/__init__.py:
```diff
@@ -36,7 +36,11 @@
+import warnings
+
+with warnings.catch_warnings():
+	warnings.simplefilter("ignore")
```
Let's add a brief comment here explaining why we need to do this.

donboie commented:
* added comment in https://github.com/GafferHQ/gaffer/pull/2373/commits/2e86ddb5b0f382e6fcc622554e878c5bd17f88fa
* fixed up description in https://github.com/GafferHQ/gaffer/pull/2373/commits/c58bf9e48eec8972721e7d4c6096d793f20247d3
